website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

```diff
@@ -0,0 +1,298 @@
+---
+title: "Lint and format your code"
+id: "lint-format"
+description: Integrate with popular linters and formatters like SQL Fluff, sqlfmt, Black, and Prettier."
+sidebar_label: "Lint and format"
+tags: [IDE]
+---
+
+Enhance your development workflow by integrating with popular linters and formatters like [SQLFluff](https://sqlfluff.com/), [sqlfmt](http://sqlfmt.com/), [Black](https://black.readthedocs.io/en/latest/), and [Prettier](https://prettier.io/). Leverage these powerful tools directly in the dbt Cloud IDE without interrupting your development flow.
+
+<details>
+<summary>What are linters and formatters? </summary>
+Linters analyze code for errors, bugs, and style issues, while formatters fix style and formatting rules. 
+</details>
+
+
+In the dbt Cloud IDE, you have the capability to perform linting and formatting on five different file types (SQL, YAML, Markdown, JSON, and Python):
+ 
+- SQL &mdash; [Lint](#lint) and fix with SQLFluff, and [format](#format) with sqlfmt
+- YAML, Markdown, and JSON &mdash; Format with Prettier
+- Python &mdash; Format with Black
+
+In the dbt Cloud IDE, each file type has its own unique linting and formatting rules. You have the option to [customize](#customize-linting) the linting process to add more flexibility and enhance problem/style detection. 
```

Comment:
It's a little repetitive; we covered that we're in the IDE in the opening statement
```suggestion
Each file type has its own unique linting and formatting rules. You can [customize](#customize-linting) the linting process to add more flexibility and enhance problem and style detection. 
```